### PR TITLE
RBAC deployment complete

### PR DESCRIPTION
**Pull request by deaborch**
Deploys Tomcat within a RBAC permissions in Kuberentes

### CODE REVIEW

File: applications/tomcat/deploy.sh [hunk 1]
```diff
@@ -50,62 +52,131 @@ touch $LOG_FILENAME
     APPLICATION_NAME="tomcat"
     TEST_FOLDER="/home/$USER_NAME/$APPLICATION_NAME"
     APPLICATION_FOLDER="applications/common"
-    
+    TEMPLATE_FOLDER="applications/tomcat"
+    NAMESPACE="tomcat"
```
Comment (rjaini) [inline applications/tomcat/deploy.sh]: Can we have a different namespace name like namespace-tomcat or ns-tomcat.

File: applications/tomcat/deploy.sh [hunk 2]
```diff
@@ -50,62 +52,131 @@ touch $LOG_FILENAME
...
+    KUBE_CERT_LOCATION="/etc/kubernetes/certs"
+    CREDENTIAL_NAME="tomcat-credentials"
+    CLUSTER_USERNAME="tomcat"
```
Comment (rjaini) [inline applications/tomcat/deploy.sh]: Can we also have different cluster name like tomcat-user or tomcatuser.

File: applications/tomcat/deploy.sh (hunk 1 above)
Comment (deaborch) [reply applications/tomcat/deploy.sh]: Done

File: applications/tomcat/deploy.sh (hunk 2 above)
Comment (deaborch) [reply applications/tomcat/deploy.sh]: done

**Pull request merged**